code/src/components/legacy/Albums.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import data from '../data.json'
+import heart from '../icons/heart_white.svg'
+import play from '../icons/play_white.svg'
+import dots from '../icons/dots_white.svg'
+
+let albums = data.albums.items.filter((album) => album.album_type === "album")
+// console.log(albums)
```

Comment:
maybe it would be a good idea to remove the console.log, before end product. Just a thought